modules/optimization/src/reporters/OptimizationData.C

```diff
@@ -315,5 +322,16 @@ OptimizationDataTempl<T>::errorCheckDataSize()
                ").");
 }
 
+template <typename T>
+Real
+OptimizationDataTempl<T>::computeMisfitNorm()
+{
+  Real val = 0.0;
+  for (auto & misfit : _misfit_values)
+    val += misfit * misfit;
+
+  return val * 0.5;
```

Comment:
this isnt a norm? you need the sqrt to make it a norm

Reply:
Changed the name of the function to `computeMisfitValue`.